templates/register.twig

```diff
@@ -0,0 +1,204 @@
+    <link rel="stylesheet" type="text/css" href="//unpkg.com/vuetify/dist/vuetify.min.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, minimal-ui">
+    <style>
+        .spinning {
+            animation-name: spin;
+            animation-duration: 5000ms;
+            animation-iteration-count: infinite;
+            animation-timing-function: linear;
+        }
+
+        @keyframes spin {
+            from {
+                transform: rotate(0deg);
+            }
+            to {
+                transform: rotate(360deg);
+            }
+        }
+    </style>
+</head>
+<body>
+<div id="app">
+    <div class="input">
+        <v-app id="setup">
+            <v-content>
+                <v-container class="fill-height" fluid>
+                    <v-row align="center" justify="center">
+                        <v-col cols="12" sm="8" md="6">
+                            <v-card class="elevation-12">
+                                <v-toolbar color="primary" dark flat>
+                                    <v-toolbar-title>GameTactic</v-toolbar-title>
+                                </v-toolbar>
+                                <router-view></router-view>
+                            </v-card>
+                        </v-col>
+                    </v-row>
+                </v-container>
+            </v-content>
+        </v-app>
+    </div>
+</div>
+<script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+<script src="//unpkg.com/vue/dist/vue.js"></script>
+<script src="//unpkg.com/vuetify/dist/vuetify.min.js"></script>
+<script src="https://unpkg.com/vue-router/dist/vue-router.js"></script>
+<!--suppress ES6ModulesDependencies -->
+<script>
+    const Register = {
+        template: `
+        <div id="check-update">
+            <v-card-text>
+                <template v-if="'{{ error }}' === ''">
+                     <v-stepper v-model="slide">
+                        <v-stepper-header>
+                            <v-stepper-step :complete="slide > 1" step="1"> Welcome </v-stepper-step>
+                            <v-stepper-step :complete="slide > 2" step="2"> Terms of Service </v-stepper-step>
+                            <v-stepper-step :complete="slide > 3" step="3"> Profile </v-stepper-step>
+                            <v-stepper-step :complete="slide > 4" step="4"> Go! </v-stepper-step>
+                        </v-stepper-header>
+                    </v-stepper>
+                    <v-divider style="margin: 15px 0 15px 0"/>
+                    <v-form>
+                        <template v-if="slide === 1">
+                            <h2>Welcome, {{ username }}!</h2>
+                            <br>
+                            <p>
+                                We need to check few things before we can continue.
+                                Some are standard things, some are customizations for your account. <br> <br>
```

Comment:
British English please :P

Reply:
Dont see difference. This is Finnish english